### PR TITLE
[Form] Add missing vietnamese translations

### PR DESCRIPTION
**Pull request by jschaedl**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | - 
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/Form/Resources/translations/validators.vi.xlf [hunk 1]
```diff
@@ -18,6 +18,102 @@
...
+                <source>The collection is invalid.</source>
+                <target>Danh sách không hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="103">
+                <source>Please select a valid color.</source>
+                <target>Vui lòng chọn một màu hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="104">
+                <source>Please select a valid country.</source>
+                <target>Vui lòng chọn một nước hợp lệ.</target>
```
Comment (van2412) [inline src/Symfony/Component/Form/Resources/translations/validators.vi.xlf:39]:
```suggestion
                <target>Vui lòng chọn đất nước hợp lệ.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.vi.xlf [hunk 2]
```diff
@@ -18,6 +18,102 @@
...
+                <source>Please select a valid language.</source>
+                <target>Vui lòng chọn ngôn ngữ hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="113">
+                <source>Please select a valid locale.</source>
+                <target>Vui lòng chọn miền hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="114">
+                <source>Please enter a valid money amount.</source>
+                <target>Vui lòng nhập số tiền hợp lệ.</target>
```
Comment (van2412) [inline src/Symfony/Component/Form/Resources/translations/validators.vi.xlf:79]:
```suggestion
                <target>Vui lòng nhập một khoảng tiền hợp lệ.</target>
```

File: src/Symfony/Component/Form/Resources/translations/validators.vi.xlf [hunk 3]
```diff
@@ -18,6 +18,102 @@
...
+                <source>The password is invalid.</source>
+                <target>Mật khẩu không hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="117">
+                <source>Please enter a percentage value.</source>
+                <target>Vui lòng nhập một giá trị phần trăm.</target>
+            </trans-unit>
+            <trans-unit id="118">
+                <source>The values do not match.</source>
+                <target>Những giá trị không phù hợp.</target>
```
Comment (van2412) [inline src/Symfony/Component/Form/Resources/translations/validators.vi.xlf:95]:
```suggestion
                <target>Các giá trị không phù hợp.</target>
```

**Review by van2412 — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @jschaedl.

**Pull request merged**